docs/docs/widgets/timer.md

```diff
@@ -2,90 +2,94 @@
 id: timer
 title: Timer
 ---
+
 # Timer
 
-The **Timer** widget lets users to count timer both upward and downward
+The **Timer** component allows users to track time by counting both upward and downward. It's useful for tasks like setting countdowns, tracking elapsed time, or timing events
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Properties
 
-| <div style={{ width:"100px"}}> Property </div>     | <div style={{ width:"100px"}}> Description </div> |
-|:----------- |:----------- | 
-| Default value | It specifies the initial value of timer. Format is: `HH.mm.ss.SS`. |
-| Timer type | It specifies If its a upward or downward counter. Select `Count Up` or `Count Down` from the dropdown or you can click on `Fx` to programmatically define the values `countUp` or `countDown`. |
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div>                                                                                                                                              |
+| :--------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Default value                                  | It specifies the initial value of timer. Format is: `HH.mm.ss.SS`.                                                                                                                             |
+| Timer type                                     | It specifies If its a upward or downward counter. Select `Count Up` or `Count Down` from the dropdown or you can click on `Fx` to programmatically define the values `countUp` or `countDown`. |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Events
 
-| <div style={{ width:"100px"}}> Event </div>     | <div style={{ width:"100px"}}> Description </div> |
-|:----------- |:----------- | 
-| On start | This event is fired when user clicks on start button. |
-| On resume | This event is fired when user clicks on resume button. |
-| On pause | This event is fired when user clicks on pause button. |
-| On count down finish | This event is fired when the count down timer reaches zero. |
-| On reset | This event is fired when user clicks on reset button. |
+| <div style={{ width:"100px"}}> Event </div> | <div style={{ width:"100px"}}> Description </div>                                                                 |
+| :------------------------------------------ | :---------------------------------------------------------------------------------------------------------------- |
```

Comment:
Hey @krishnadev7, can you please recheck this table?
The events and their descriptions do not match. Also, this is the timer component, you have added description for the text input component, please correct that too.
Thanks!